src/index.html

```diff
@@ -9,9 +9,191 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="label">
+          Surname:
+
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+            pattern="\w+"
```

Comment:
```suggestion
            pattern="\w+"
```
Do you really need it there? Describe why you need it or remove